spec/formatting.md

```diff
@@ -85,10 +85,16 @@ At a minimum, it includes:
 
 Implementations MAY include additional fields in their _formatting context_.
 
-## Expression and Markup Resolution
+## Resolved Values
 
-_Expressions_ are used in _declarations_, _selectors_, and _patterns_.
-_Markup_ is only used in _patterns_.
+This specification allows for the same value to be used for:
+- formatting in a _placeholder_,
+- selection in a _selector_ _expression_,
+- as the _operand_ of another _expression_ (including _local declarations_), or
+- as an _option_ value in another _expression_,
```

Comment:
```suggestion
This specification allows for the same interface to be used as:
- a result of evaluating a _placeholder_, before it's formatted,
- a result of evaluating a _selector_ _expression_, before it's selected on,
- as the _operand_ of another _expression_ (including _local declarations_), or
- as an _option_ value in another _expression_,
```

Reply:
The intent with this paragraph is to communicate about end user capabilities, not about internal implementation details. So this is about being able to use the same value (say, a number passed into the implementation's formatting method as an argument) for multiple purposes.

In the syntax, this is effected by _variable references_, _declarations_, and other syntax features, and in the spec's runtime definition, by _resolved values_.

This proposed change flips the meaning of the text by discussing internal rather than external use cases. It also introduced the term "interface", which further increases the distance from concrete use cases, as well as rooting what follows in an object-oriented approach. This isn't invalid by any means, but it's presuming the reader to have a lot more context than the current text.

Reply:
Closing as out-of-date.